skyportal/handlers/api/gcn.py

```diff
@@ -390,29 +390,25 @@ def post_skymap_from_notice(
                         'ra': ra,
                         'dec': dec,
                     }
-                elif any(
-                    [True if 'GRB' in tag.text.upper() else False for tag in tags]
-                ):
+                elif any([True if 'GRB' in tag.upper() else False for tag in tags]):
```

Comment:
why is `.text` not needed? Have the tags been converted to text somewhere above? Is that true for all notices (and not just JSON)?

Reply:
yeah these tags are all just the plain text.

Reply:
This is only the case where its like gcnevent.tags.

Reply:
got it, yeah it was just confusing that .text all of a sudden had to go away but nothing else upstream of that seems to have changed.